src/budy/models/bundle.py

```diff
@@ -194,8 +194,8 @@ def add_product_s(
             _line = line
 
         if _line:
-            if not increment: return _line
-            _line.quantity += quantity
+            if increment: _line.quantity += quantity

```

Comment:
To be able to update a line without incrementing. Returning when quantity remains the same should solve the problem if the overall approsch is correct.